Makefile.toml

```diff
@@ -46,13 +47,35 @@ alias = "dev-flow"
 [tasks.dev-flow]
 description = "Runs a full development flow, including fixing format and clippy, building and running tests"
 dependencies = [
+    "wit",
     "fix",
     "check",
     "build",
     # "test"
 ]
 
+# WIT DEPENDENCIES
+[tasks.wit]
+description = "Fetches the WIT dependencies based on wit/deps.toml"
+dependencies = ["wit-host", "wit-wasm-rpc"]
+
+[tasks.wit-wasm-rpc]
+dependencies = ["wit-host"]
+script_runner = "@duckscript"
+script = """
+rm -r wasm-rpc/wit/deps
```

Comment:
do we really want to delete and re-get the deps on basically every command? (i know that wit-deps uses a cache, and probably not a huge performance problem, still feels wasteful)

Reply:
This is not "regetting" them just copying them from the project root's `wit/deps`. We can try to make a better script for it but
- it needs to be working on all platforms - that's why i used this 🦆 thing
- it needs to remove things that are no longer there - i already ran into that multiple times as i was trying to figure out the proper name for the new wit package

Reply:
by "regetting" i meant wit-host also runs on everything now, and yes, i understand that it would need a "task checker", will think about it (ironically golem-cli app could do it :D)

Reply:
The `wit-dep update` is designed to be runnable every time. That's why it keeps a lock file.

Reply:
(Based on https://github.com/bytecodealliance/wit-deps/issues/202 being closed)